stochatreat/stochatreat.py

```diff
@@ -137,24 +137,26 @@ def stochatreat(data: pd.DataFrame,
     # sort data
     data = data.sort_values(by=idx_col)
 
-    # combine block cells
-    data = data[[idx_col] + stratum_cols].copy()
-    data['stratum'] = data[stratum_cols].astype(str).sum(axis=1)
-    strata = sorted(set(data['stratum']))
+    # combine block cells - by assigning stratum ids
+    data['stratum_id'] = data.groupby(stratum_cols).ngroup()
+    strata = range(data['stratum_id'].max() + 1)
```

Comment:
`data['stratum_id'].unique()` would do the trick as well, no?

Reply:
Yup, better